### DISCUSSION TITLE
How to change permissions form the docker output?

### DISCUSSION THREAD

**Discussion by T0biasKl** [dita-ot/dita-ot#4273] [Q&A]
I'm new to Docker and Dita and tried to generate an output using Docker and Dita 4.1

With
```shell
podman run -it  -v ~/source/mobile-phone:/src ghcr.io/dita-ot/dita-ot:4.1 -i /src/mobilePhone.ditamap -o /src/out -f html5 -v
```
I get an output in the mounted directory but they have restricted permissions. 
I know that the docker is the author but how can I change the output so that it has no restriction?

**Comment by jelovirt** [comment]
@T0biasKl Could you add a little more details? For example show `ls -l` output for `~/source/mobile-phone/out`.

**Reply by T0biasKl** [reply]
The `~/source/mobile-phone/out` folder does not exist and should be generated by via the dita output. Changed recursive the permissions of `~/source/` and therefor had no access. 
TL: Had a stupid permission problem not a dita or docker problem. the above command works as intended.
